### PR TITLE
Add missing functions for templates and bulking

### PR DESCRIPTION
**Pull request by thermech**
- Add create_template
- Add delete_template
- Add send_bulk_templated_email

### CODE REVIEW

File: lib/ex_aws/ses.ex [hunk 1]
```diff
@@ -212,13 +265,46 @@ defmodule ExAws.SES do
     )
   end
 
-  defp format_dst(dst) do
+  defp format_dst(dst, root \\ "destination") do
```
Comment (lessless) [inline lib/ex_aws/ses.ex:268]: Nice, thanks!

File: test/lib/ses_test.exs [hunk 2]
```diff
@@ -136,6 +136,101 @@ defmodule ExAws.SESTest do
     end
   end
 
+  describe "#send_bulk_templated_email" do
+    test "with required params only" do
+      template = "my_template"
+      source = "user@example.com"
+
+      replacement_template_data1 = %{data1: "value1"} |> Poison.encode!()
```
Comment (lessless) [inline test/lib/ses_test.exs]: If those values will always be encoded, then I guess it makes sense to encode them inside a `send_bulk_templated_email/3`

**Review by lessless — CHANGES_REQUESTED** [review]
Hi @thermech ,

Thanks a lot for a massive contribution. Have one question about and ready to merge & release new version.

Have a great weekend!

File: test/lib/ses_test.exs (hunk 2 above)
Comment (thermech) [reply test/lib/ses_test.exs]: Yeah, good Idea!

**Comment by thermech** [comment]
@lessless I've apply your suggestions. Thx

**Review by lessless — APPROVED** [review]

**Comment by lessless** [comment]
thanks @thermech !

**Comment by lessless** [comment]
@thermech available as 2.1.0

**Comment by thermech** [comment]
🎉 Perfect, thanks!

**Pull request merged**